app/safe/internal/bootstrap/bootstrap.go

```diff
@@ -59,36 +59,36 @@ func Monitor(
 		select {
 		// Acquired SVID for this workload from the SPIRE Agent via workload API:
 		case <-channels.AcquiredSvid:
-			log.InfoLn(correlationId, "Acquired identity.")
+			log.AuditLn(correlationId, "Acquired identity.")
```

Comment:
`InfoLn` does not print anything with the default log level (3).

Switched to Audit to always print something regardless of the log level.

This is useful to ensure that VSecM safe initialized properly.